locust/main.py

```diff
@@ -491,20 +497,21 @@ def ensure_user_class_name(config):
         else:
             web_host = options.web_host
         if web_host:
-            logger.info(f"Starting web interface at {protocol}://{web_host}:{options.web_port}")
+            logger.info(f"Starting web interface at {protocol}://{web_host}:{options.web_port}{options.base_url}")
```

Comment:
I think `url` doesn't actually make sense in this case, URL refers to the entire composition (e.g. protocol://domain:port/path?query=params), in this case `base_path` would be more accurate